src/openrct2/object/SceneryGroupObject.cpp

```diff
@@ -192,7 +192,35 @@ std::vector<ObjectEntryDescriptor> SceneryGroupObject::ReadJsonEntries(json_t& j
 
     for (const auto& jEntry : jEntries)
     {
-        entries.emplace_back(Json::GetString(jEntry));
+        auto entryName = Json::GetString(jEntry);
+        if (String::StartsWith(entryName, "$DAT:"))
+        {
+            if (entryName.length() != 5 + 8 + 1 + 8)
```

Comment:
I'd add a comment above with an example of the format so that you can more easily connect the numbers and the parts of the string together.